packages/overlay/src/index.tsx

```diff
@@ -121,16 +142,32 @@ export async function init({
     });
   }
 
+  const tabs = initializedIntegrations
+    .map(integration => {
+      if (integration.tabs) {
+        return integration.tabs({ processedEvents: [] }).map(tab => ({
+          ...tab,
+          processedEvents: [],
+        }));
+      }
+      return [];
+    })
+    .flat();
+
+  const initialTab = tabs.length ? `/${tabs[0].id}` : '/no-tabs';
+
   ReactDOM.createRoot(appRoot).render(
     // <React.StrictMode>
-    <App
-      integrations={initializedIntegrations}
-      fullScreen={fullScreen}
-      defaultEventId={defaultEventId}
-      showTriggerButton={showTriggerButton}
-      sidecar={sidecar}
-      anchor={anchor}
-    />,
+    <MemoryRouter initialEntries={[initialTab]}>
+      <App
+        integrations={initializedIntegrations}
+        fullScreen={fullScreen}
+        defaultEventId={defaultEventId}
```

Comment:
I think we can remove this option then, right? (already marked as a removal candidate in the JSDoc)